services/src/main/scala/cromwell/services/keyvalue/KeyValueService.scala

```diff
@@ -19,4 +19,3 @@ object KeyValueService {
   case class KvKeyLookupFailed(action: KvGet) extends KvResponse
   case class KvPutSuccess(action: KvPut) extends KvResponse
 }
-
```

Comment:
Perhaps a massive side-comment: Import optimisation from IntelliJ is a nightmare waiting to happen.

IntelliJ is awful at working implicit imports backwards and I had to turn off auto-import-correction when it trashed all of the Slick JSON implicit imports and I spent hours trying to determine which combination of imports was required in all the files which were "corrected"